openms-wiki/developer-faq.md

```diff
@@ -0,0 +1,490 @@
+Developer FAQ
+=============
+
+The following contains answers to typical questions from developers about OpenMS.
+
+> **_NOTE:_**  Some actions require special permissions like e.g. updating the website.
+
+## General
+
+The following section provides general information to new contributors.
+
+### I am new to OpenMS. What should I do first?
+
+* Check out the development version of OpenMS (see website).
+* Try and build OpenMS according to the installation instructions.
+* Read the OpenMS Coding Convention.
+* Read the OpenMS Tutorial.
+* Create a GitHub account
+* Register to the open-ms-general and open-ms-developers mailing list. (You can see the developers list only if you are logged in to Sourceforge and if you are a OpenMS developer).
```

Comment:
> Register to

Subscribe to the [open-ms-general]() and [open-ms-developers]() mailing lists.

within a note: (You can see the developers list only if you are logged in to Sourceforge and if you are a OpenMS developer).

Reply:
can you post the link to the open-ms-developers mailing list? I've only got the general one.